src/components/Alerts/ResumeDialog.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import { Tenant } from "@/types/tenants";
+import Button from "../button/Button";
+import { AlertDialogDescription, AlertDialogTitle, AlertDialogContent } from "./AlertDialog";
+import { Action, Cancel } from "@radix-ui/react-alert-dialog";
+import { apiRoutes, getAPIURL } from "@/data/api";
+import { useRouter } from "next/navigation";
+import { ComponentPropsWithoutRef, ElementRef, forwardRef, useState } from "react";
+import { routes } from "@/routes";
+
+type ResumeDialogProps = {
+	id: Tenant["id"];
+};
+
+const ResumeDialog = forwardRef<
+	ElementRef<typeof AlertDialogContent>,
+	ComponentPropsWithoutRef<typeof AlertDialogContent> & ResumeDialogProps
+>(({ id }, ref) => {
+	const [submitted, setSubmitted] = useState(false);
+	const router = useRouter();
```

Comment:
useRouter is `next` specific. We should try to be agnostic, so we can use the package with `react-router-dom` as well